ui/packages/atlasmap/src/Atlasmap/useContextToolbar.tsx

```diff
@@ -58,13 +61,15 @@ export function useContextToolbar({
   showColumnMapperViewToolbarItem = true,
   showMappingTableViewToolbarItem = true,
   showNamespaceTableViewToolbarItem = true,
+  showTransformationApproachViewToolbarItem = true,
```

Comment:
I think this should be `false` for the moment since the view is still in progress, else it would be displayed to users.

Reply:
I can do that, but my thought here was to keep it true so that the team and whatever other developers can see and try out what I'm working on as I go, with the idea that I'd change it to false if things aren't far enough along by the next release.

Reply:
The netlify link on this PR is probably the easiest way for folks to see how this is progressing.  I'd be wary of letting partially completed features into master.  I had a peek earlier today and while I don't know what the gaps are in terms of what's completed, it also wasn't clear to me yet where the benefit is.  I see that it restricts when the mapping lines are drawn, however what's the plan on how the user edits a given mapping, or did I miss that?

Reply:
k, no problem, I'll set the flag to false. WRT to benefit of what I'm doing, I emailed a mockup to you and Riccardo a couple of weeks back that should explain the goal with this. You won't see much yet just because I've been waiting for this part to be approved and merged so I can start adding manageable reviewable commits afterwards. Let me know if you didn't receive it, and I'll resend it.

Reply:
Yep, I did get it, thanks!  I just don't understand why it's necessary to get bits into master before proceeding?

Reply:
Working on this large of a feature isn't tenable in my own private repo. I need to get feedback from the team about the additions I make along the way, so it's important that they can see those changes and try them out as I proceed. With the flag off, the code won't affect any users in any way, but will be an easy toggle for developers to turn on the feature to try it out. I, of course, also want my contributions to be evident in this repo.

Reply:
Well, the other option for a long-running thing would be a separate branch in this repo alongside master that would allow you to open PRs against.

Reply:
True, but again, no one will look at it that way. I guess I don't see the harm in putting this in master, but I'll do whatever you need me to do.